### PR TITLE
chore: trying fromkk/SpellChecker instead of ezura/spell-checker-for-swift

### PR DESCRIPTION
**Pull request by mohammedDehairy**
chore: trying fromkk/SpellChecker instead of ezura/spell-checker-for-swift

### CODE REVIEW

**Review by nauaros — APPROVED** [review]

File: spell-check-word-allow-list.yaml [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+  - rhs
+  - lhs
+  - Unkeyed
+  - psp
+  - Iterable
+  - Finalizable
+  - GitHub
+  - weixin
+  - Eldoheiri
+  - Abramichev
```
Comment (descorp) [inline spell-check-word-allow-list.yaml:81]: we can probably just fix headers 😆

File: spell-check-word-allow-list.yaml [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+  - eftpos
+  - nanaco
+  - waon
+  - deinit
+  - darwin
+  - cyclomatic
+  - doesn
+  - couldn
+  - brazilian
+  - australian
```
Comment (descorp) [inline spell-check-word-allow-list.yaml:162]: What is wrong here? Should be capital?

File: spell-check-word-allow-list.yaml (hunk 1 above)
Comment (mohammedDehairy) [reply spell-check-word-allow-list.yaml:81]: yeah, lets do that in another PR :-)

File: spell-check-word-allow-list.yaml (hunk 2 above)
Comment (mohammedDehairy) [reply spell-check-word-allow-list.yaml:162]: its a quirk of the spell check tool, some how it thinks `Australian` and `Brazilian` are typos 🤷‍♂️

**Review by descorp — APPROVED** [review]

**Pull request merged**